apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2/BigQueryClientImpl.InsertData.cs

```diff
@@ -136,39 +135,20 @@ private BigQueryJob UploadData(JobConfigurationLoad loadConfiguration, Stream in
         }
 
         /// <inheritdoc />
-        public override void InsertRows(TableReference tableReference, IEnumerable<BigQueryInsertRow> rows, InsertOptions options = null)
+        public override BigQueryInsertResults InsertRows(TableReference tableReference, IEnumerable<BigQueryInsertRow> rows, InsertOptions options = null)
         {
             var request = CreateInsertAllRequest(tableReference, rows, options, out bool hasRows);
```

Comment:
This is where I'm expecting something will be creating a List<BigQueryInsertRow>` internally. We may be able to be more efficient. I'll look once most of this is in, as it'll be simpler to work out then.